src/alert/src/InlineAlert.js

```diff
@@ -1,36 +1,51 @@
 import React, { memo, forwardRef } from 'react'
+import cx from 'classnames'
 import PropTypes from 'prop-types'
 import { spacing, dimensions, position, layout } from 'ui-box'
 import { Pane } from '../../layers'
 import { Text } from '../../typography'
-import { useTheme } from '../../theme'
+import useStyleConfig from '../../hooks/use-style-config'
 import { getIconForIntent } from './getIconForIntent'
 
+const pseudoSelectors = {}
+
+const internalStyles = {
+  display: 'flex',
+  alignItems: 'center'
+}
+
 const InlineAlert = memo(
   forwardRef(function InlineAlert(props, ref) {
     const {
       children,
-      intent = 'none',
+      className,
+      intent = 'info',
       hasIcon = true,
       size = 400,
       ...restProps
     } = props
 
-    const {
-      tokens: { intents }
-    } = useTheme()
-
     const intentToken = intent === 'none' ? 'info' : intent
-    const textColor = intents[intentToken].text
+    const { className: themedClassName, ...styleProps } = useStyleConfig(
+      'InlineAlert',
+      { intent: intentToken },
```

Comment:
Feels weird to support `intent="none"` and `intent="info"` and have them map to the same thing, no?

Reply:
100% agree. Looks like v5 and earlier supported `intent="none"` but internally treated "none" the same as "info".

We could drop it, but it will require rewriting a bunch of components to use `intent="info"` (or omit the intent)